akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -360,7 +361,8 @@ class HttpExt private[http] (private val config: Config)(implicit val system: Ex
     remoteAddress:      Option[InetSocketAddress] = None,
     log:                LoggingAdapter            = system.log,
     isSecureConnection: Boolean                   = false): ServerLayer = {
-    val server = HttpServerBluePrint(settings, log, isSecureConnection)
+    val parsingErrorHandler = system.dynamicAccess.createInstanceFor[ParsingErrorHandler](settings.parsingErrorHandler, Nil).get
```

Comment:
I guess conceptionally, this could also go into the settings class but if it's only used here, I guess it's fine to keep it here.